### DISCUSSION TITLE
Bravo opens for a second and closes

### DISCUSSION THREAD

**Discussion by 7naomi** [sql-bi/Bravo#802] [Q&A]
Newbie here!   I have installed bravo however when I click on it the screen opens for a moment and then just closes.  Does anyone one know what I am doing incorrectly.

Thank you!

**Comment by albertospelta** [comment]
Hi @7naomi. Could you let us know which version of Windows you’re using and whether the app is running on a local PC or on a server  ? (e.g., via remote desktop, terminal server, Citrix)

It looks like there might be an issue causing the app to crash right at startup. Have you checked the Windows Event Log for any error or warning messages around the time you attempted to open the app?
